libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -414,6 +427,121 @@ void AP_Mount_Backend::write_log(uint64_t timestamp_us)
     AP::logger().WriteCriticalBlock(&pkt, sizeof(pkt));
 }
 
+#if AP_MOUNT_POI_TO_LATLNGALT_ENABLED
+// get poi information
+bool AP_Mount_Backend::get_poi(uint8_t instance, Quaternion &quat, Location &loc, Location &poi_loc)
```

Comment:
```suggestion
bool AP_Mount_Backend::get_poi(uint8_t instance, Quaternion &quat, Location &loc, Location &poi_loc) const
```

Reply:
I think this cant be done cause its using a semaphore here. See `error: binding reference of type 
 AP_HAL::Semaphore&' to 'const AP_HAL::Semaphore' discards qualifiers
  435 |         WITH_SEMAPHORE(poi_calculation.sem);`

Reply:
Ah, darn.  Worth a shot, I guess.